diffmatchpatch/diff_test.go

```diff
@@ -998,8 +998,38 @@ func TestDiffText(t *testing.T) {
 }
 
 func TestDiffDelta(t *testing.T) {
+
+	type TestCase struct {
+		Text               string
```

Comment:
Please add a "Name" field too and use the comments for the test case values. You can pretty much copy the pattern (e.g. for the error messages for the asserts) for that from other test functions.